client.go

```diff
@@ -207,6 +207,18 @@ var clientOptionsParsers = map[string]parserFunc{
 
 		return nil
 	},
+	"enableTracing": func(config *ClientConfig, enableTracing string, _ ...string) error {
```

Comment:
```suggestion
	"tracking": func(config *ClientConfig, enableTracing string, _ ...string) error {
```
`tracing` naming can be confused with stuff related to stacktraces or tracing engines